user_news/forms.py

```diff
@@ -12,18 +12,13 @@ class Meta:
 
 class OutageForm(ModelForm):
     send_email_notification = BooleanField(required=False)
-    send_email_reminder = BooleanField(required=False)
 
     class Meta:
         model = Outage
+        exclude = ['reminder_sent']
```

Comment:
Nice, didn't know about this thing. https://docs.djangoproject.com/en/1.11/topics/forms/modelforms/#selecting-the-fields-to-use